modules/gitops-release-notes-1-6-0.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-6-0_{context}"]
+= Release notes for {gitops-title} 1.6.0
+
+{gitops-title} 1.6.0 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="new-features-1-6-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* Previously, the Argo CD `ApplicationSet` controller was a technology preview (TP) feature. With this update, it is a general availability (GA) feature. link:https://issues.redhat.com/browse/GITOPS-1958[GITOPS-1958]
+
+* With this update, the latest releases of the {gitops-title} are available in `latest` and version-based channels. To get these upgrades, update the `channel` parameter in the `Subscription` object YAML file: change its value from `stable` to `latest` or a version-based channel such as `gitops-1.6`. link:https://issues.redhat.com/browse/GITOPS-1791[GITOPS-1791]
```

Comment:
Please reevaluate this statement: "With this update, the latest releases of the {gitops-title} are available in `latest` and version-based channels."